src/spas/libs/member-management/feature/src/lib/member-management/users/users.component.html

```diff
@@ -0,0 +1,8 @@
+<h2>Users</h2>
+<ul>
+  <li *ngFor="let user of users | async" (click)="onSelect(user)">
+    {{user.fullName}}
+  </li>
+</ul>
+
+<jwadmin-user-detail [user]="selectedUser"></jwadmin-user-detail>
```

Comment:
selector for feature lib should start with the name of the feature like ```<member-management-user-detail>```, ```jwadmin``` can be removed

Reply:
I think the CLI did this too. For some reason it automatically creates it with the jwadmin selector already. Any way to specify the selector name when you create a new component using the CLI?